packages/core/src/components/tag-input/tagInput.tsx

```diff
@@ -15,6 +15,17 @@ import * as Utils from "../../common/utils";
 import { Icon, IconName } from "../icon/icon";
 import { ITagProps, Tag } from "../tag/tag";
 
+/**
+ * A type reflecting the manner in which a `TagInput` value was added.
+ * - `"default"` - indicates that a value was added in the default fasion, via
```

Comment:
nit: "fashion"

actually I would just simplify, remove the metaphor, and say "... was added by manual selection"